src/apps/explorer/components/common/Search/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useMemo } from 'react'
+import styled from 'styled-components'
+import SVG from 'react-inlinesvg'
+import { useHistory, useRouteMatch } from 'react-router-dom'
+
+// assets
+import searchImg from 'assets/img/search2.svg'
+
+const Wrapper = styled.form`
+  display: flex;
+  width: 100%;
+  max-width: 50rem;
+  margin: 0 auto;
+  position: relative;
+
+  > input {
+    height: 5rem;
+    flex: 1 1 auto;
+    background: ${({ theme }): string => theme.greyOpacity};
+    font-weight: ${({ theme }): string => theme.fontMedium};
+    font-size: 1.6rem;
+    border-radius: 0.6rem;
+    line-height: 1;
+    display: flex;
+    outline: 0;
+    appearance: none;
+    align-items: center;
+    color: ${({ theme }): string => theme.greyShade};
+    padding: 1.6rem 1.6rem 1.6rem 5rem;
+    box-sizing: border-box;
+    border: 0.1rem solid transparent;
+    transition: border 0.2s ease-in-out;
+
+    &:focus {
+      border: 0.1rem solid ${({ theme }): string => theme.borderPrimary};
+    }
+
+    &::placeholder {
+      color: inherit;
+      transition: color 0.2s ease-in-out;
+    }
+
+    &:focus::placeholder {
+      color: transparent;
+    }
+  }
```

Comment:
I think this could be a separated styled component. Feels too big IMO.

Reply:
Agreed, will separate!